src/main/java/frc/robot/commands/ToggleIntake.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Intake;
+import frc.robot.Robot;
+
+public class ToggleIntake extends CommandBase {
+    private Intake intake;
+
+    // Constructor
+    public ToggleIntake(Intake intake) {
+        addRequirements(intake);
```

Comment:
I would remove passing in the Intake as an argument.
A call to `Robot.intake` would be better and more central.
(Meaning you'll also need to remove like 8, and replace this.intake to the above.)